internal/common/rules/globalvariableregistration.go

```diff
@@ -30,4 +30,48 @@ var StaticVariables = []VariableValuePair{
 		VariableName: "max_severity_of_sanctions",
 		Values:       []float64{2},
 	},
+	{
+		VariableName: "no_islands_alive",
+		Values:       []float64{6},
```

Comment:
Maybe take the number of islands in lines like these from a variable instead? @QFSW has also mentioned in another PR that we may want to be able to extend to more than 1 island per team in the future. While unlikely, it's not a very big change that may go a long way :)

Reply:
go ahead and use `len(shared.TeamIDs)`